tests/nlu/featurizers/test_convert_featurizer.py

```diff
@@ -10,7 +10,7 @@
 from rasa.nlu.featurizers.dense_featurizer.convert_featurizer import ConveRTFeaturizer
 
 
```

Comment:
can you please add a comment and a reference to the issue why this test is skipped?